cmd/discord_bot/main.go

```diff
@@ -5,59 +5,24 @@ import (
 
 	"github.com/caarlos0/env/v11"
 	environment "github.com/ccil-kbw/robot/internal/environment"
+	logger "github.com/ccil-kbw/robot/internal/logger"
 	"github.com/ccil-kbw/robot/pkg/discord"
-	"go.uber.org/zap"
-)
-
-// TODO: Should Types have a folder of their own?
-type config struct {
-	Environment        string `env:"ENVIRONMENT" envDefault:"dev"`
-	DiscordServerID    string `env:"DISCORD_SERVER_ID"`
-	DiscordBotToken    string `env:"DISCORD_BOT_TOKEN"`
-	DiscordBotAsPublic bool   `env:"DISCORD_BOT_AS_PUBLIC" envDefault:"true"`
-}
-
-var (
-	cfg    = config{}
-	logger *zap.Logger
 )
 
 func init() {
 	environment.LoadEnvironmentVariables()
+	logger.InitializeLogger()
 	loadConfig()
-	initializeLogger()
 }
 
 func main() {
-	bot := discord.NewDiscordBot(logger, cfg.DiscordServerID, cfg.DiscordBotToken, cfg.DiscordBotAsPublic)
+	bot := discord.NewDiscordBot(logger.Logger, logger.Cfg.DiscordServerID, logger.Cfg.DiscordBotToken, logger.Cfg.DiscordBotAsPublic)
```

Comment:
J'aime pas que jdois call logger et ensuite call Logger deux fois.

Reply:
How about we name our logger module just `log`, this will be `log.Logger`?

Reply:
I thought making the var private and using a getter instead.

Reply:
Jpense jvai laisser "as is" for now. LMK ce que t'en penses.

Reply:
Looks good as is I agree 💯